src/App/Hotkeys.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  import * as modal from "@app/lib/modal";
+
+  import ColorPalette from "./ColorPalette.svelte";
+  import HelpModal from "./HelpModal.svelte";
+
+  const onKeydown = (event: KeyboardEvent) => {
+    if (event.key === "Escape") {
+      modal.hide();
+      return;
+    }
+
+    switch (event.key) {
+      case "?":
+        modal.toggle(HelpModal, {});
+        break;
+      case "/":
+        event.preventDefault();
+        (
+          document.querySelector(
+            '*[placeholder="Search a name or address…"]',
+          ) as HTMLElement | null
+        )?.focus();
```

Comment:
Using a variable within the `case` upsets our linter. 
```
/Users/rudolfs/work/radicle-interface/src/App/Hotkeys.svelte
  19:9  error  Unexpected lexical declaration in case block  no-case-declarations
  ```
We could move it outside of the `case` block, but then we'd do it on every keypress, that's why i code-golfed it inside in the first place.

Would adding a comment help maybe?

Reply:
I think you should be able to fix the linter error by encapsulating the case block in brackets like 

```ts
    switch (event.key) {
      case "?":
        modal.toggle(HelpModal, {});
        break;
      case "/": {
        event.preventDefault();
        const searchInput: HTMLElement | null = document.querySelector(
          '*[placeholder="Search a name or address…"]',
        );
        searchInput?.focus();
        break;
      }
      case "d":
        if (import.meta.env.PROD) {
          return;
        }
        modal.toggle(ColorPalette, {});
        break;
    }
```

Reply:
Good point. 58bd8816fdd5e400fc179263540299848dbca245